stonesoup/plotter.py

```diff
@@ -1508,6 +1508,50 @@ def plot_sensors(self, sensors, mapping=[0, 1], sensor_label="Sensors", **kwargs
         sensor_xy = np.array([sensor.position[mapping, 0] for sensor in sensors])
         self.fig.add_scatter(x=sensor_xy[:, 0], y=sensor_xy[:, 1], **sensor_kwargs)
 
+    def plot_obstacles(self, obstacles, mapping=[0, 1], obstacle_label='obstacles', **kwargs):
```

Comment:
```suggestion
    def plot_obstacles(self, obstacles, mapping=[0, 1], label='obstacles', **kwargs):
```
Formally, I would have suggested `obstacle_label` but due to #1071 `label` is all that is required. The format of labels has changed by removing the prefix to "_label", i.e., `###_label` -> `label`.